src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -170,6 +197,8 @@ function useChatUtilsHook(): UseChatUtilsHook {
         isLoadingFetchingChannelDetail: false
       }));
       refresh('channelInfo');
+      refresh('channelList');
+      refresh('chats');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refresh` function is now being called with additional arguments ('channelList' and 'chats'). Ensure that these new arguments are valid and that the `refresh` function can handle them appropriately.

```diff
  refresh('channelInfo');
+ refresh('channelList');
+ refresh('chats');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->